lemur/plugins/lemur_aws/iam.py

```diff
@@ -38,24 +38,68 @@ def retry_throttled(exception):
 def get_name_from_arn(arn):
     """
     Extract the certificate name from an arn.
+    examples:
+    'arn:aws:iam::123456789012:server-certificate/example.com'
+    'arn:aws:iam::123456789012:server-certificate/cloudfront/example.com-cloudfront'
+    'arn:aws:acm:us-west-2:123456789012:certificate/example.com'
+
 
     :param arn: IAM SSL arn
     :return: name of the certificate as uploaded to AWS
     """
-    return arn.split("/", 1)[1]
+    return arn.split("/")[-1]
```

Comment:
this allows the name to be without the path, as the path doesn't help with finding the certname in Lemur, as done in the next step